app/assets/stylesheets/style_updates.css.scss

```diff
@@ -60,21 +61,18 @@ html {
   height:100%;
   top:50px;
   left:0px;
-  font-family: 'Tangerine', cursive;
   .container {
     padding-left: 28vw;
     padding-top: 10vw;
-  }
-
-  .form {
-    max-width: 200px;
-
-    .label{
-      font-size: 35px;
-      color: #000000;
+      .form {
+        max-width: 200px;
+        .label{
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 4<br>Opening curly brace `{` should be preceded by one space